src/infra/repos/postgresql/breed-repository.ts

```diff
@@ -11,15 +11,17 @@ export class BreedRepository implements LoadBreedByNameRepository, LoadCatBreeds
 
   async loadCatBreeds (): Promise<LoadCatBreedsRepository.Result> {
     const catBreeds = await db.breed.findMany({ where: { specie: { name: 'Gato' } } })
-    if (catBreeds) {
+    if (catBreeds.length > 0) {
       return catBreeds
     }
+    return []
   }
 
   async loadDogBreeds (): Promise<LoadDogBreedsRepository.Result> {
     const dogBreeds = await db.breed.findMany({ where: { specie: { name: 'Cachorro' } } })
-    if (dogBreeds) {
+    if (dogBreeds.length > 0) {
       return dogBreeds
     }
+    return []
   }
```

Comment:
I believe it can be returned directly, because if it's empty it will already return an empty array and if it has something it will return the data